api/api.go

```diff
@@ -156,7 +156,7 @@ type Config struct {
 	// TLSConfig is ignored if HttpClient is set.
 	TLSConfig *TLSConfig
 
-	Headers http.Header
+	Header http.Header
```

Comment:
Any thoughts on this rename? 
Header**s** seemed more natural to me, but the type http.Header contains the map, so, maybe Header is more correct?
Perhaps it should be Header**s** because it will go on all requests, where as Header refers to a single request. 
😵

Reply:
vault calls it headers, so I'm going with that :D